src/Streamly/Data/Serialize/Instances.hs

```diff
@@ -18,12 +18,15 @@ import Data.Scientific (Scientific)
 import Data.Time (Day, TimeOfDay, LocalTime, DiffTime, UTCTime)
 import Streamly.Data.Serialize.Instances.Text ()
 import Streamly.Internal.Data.Serialize (Serialize(..))
-
 import qualified Data.Aeson as Aeson
 import qualified Data.ByteString.Lazy as BSL
 import qualified Data.Vector as Vector
 import qualified Streamly.Internal.Data.Serialize.TH as Serialize
-
+import Data.List (foldl')
+import Data.Proxy
+import qualified Data.Vector.Mutable as MVector
+import qualified Streamly.Internal.Data.Unbox as Unbox
+import Streamly.Internal.Data.Unbox (MutableByteArray)
```

Comment:
Group the explicit imports first
then the qualified imports 
then the whole module imports